From a24c7ffa0b19855f9db75bb2414ac338247f4ad6 Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Sun, 10 Dec 2023 15:42:24 +0100
Subject: [PATCH] tests: Fix build with libxml 2.12

libxml 2.12.0 reorganized the headers, which removed `xmlParseMemory` function from the scope.

Change-Id: Ibd747fe7f83eb7c3a8b8cf57ac4a06e365986d46
---

diff --git a/src/test/importtest.cpp b/src/test/importtest.cpp
index 9a01473..4a26b4a 100644
--- a/src/test/importtest.cpp
+++ b/src/test/importtest.cpp
@@ -14,6 +14,7 @@
 
 #include <libvisio/libvisio.h>
 
+#include <libxml/parser.h>
 #include <libxml/xpath.h>
 
 #include "xmldrawinggenerator.h"
